source/atlas-cli-create-cluster-from-config-file.txt

```diff
@@ -0,0 +1,225 @@
+                 "id": "<64403dd1f2a6b45e71527d5a>",
+                 "mongoDBMajorVersion": "6.0",
+                 "mongoDBVersion": "6.0.5",
+                 "name": "mySandbox",
+                 "paused": false,
+                 "pitEnabled": true,
+                 "stateName": "IDLE",
+                 "replicationSpecs": [
+                   {
+                     "numShards": 1,
+                     "id": "64403dbb0a052449df3d04ae",
+                     "zoneName": "Zone 1",
+                     "regionConfigs": [
+                       {
+                         "analyticsAutoScaling": {
+                           "diskGB": {
+                             "enabled": true
+                           },
+                           "compute": {
+                             "enabled": true,
+                             "scaleDownEnabled": true,
+                             "minInstanceSize": "M10",
+                             "maxInstanceSize": "M40"
+                           }
+                         },
+                         "analyticsSpecs": {
+                           "diskIOPS": 3000,
+                           "ebsVolumeType": "STANDARD",
+                           "instanceSize": "M10",
+                           "nodeCount": 0
+                         },
+                         "electableSpecs": {
+                           "diskIOPS": 3000,
+                           "ebsVolumeType": "STANDARD",
+                           "instanceSize": "M10",
+                           "nodeCount": 3
+                         },
+                         "readOnlySpecs": {
+                           "diskIOPS": 3000,
+                           "ebsVolumeType": "STANDARD",
+                           "instanceSize": "M10",
+                           "nodeCount": 0
+                         },
+                         "autoScaling": {
+                           "diskGB": {
+                             "enabled": true
+                           },
+                           "compute": {
+                             "enabled": true,
+                             "scaleDownEnabled": true,
+                             "minInstanceSize": "M10",
+                             "maxInstanceSize": "M40"
+                           }
+                         },
+                         "priority": 7,
+                         "providerName": "AWS",
+                         "regionName": "US_EAST_1"
+                       }
+                     ]
+                   }
+                 ],
+                 "createDate": "2023-04-19T19:15:29Z",
+                 "rootCertType": "ISRGROOTX1",
+                 "versionReleaseSystem": "LTS",
+                 "terminationProtectionEnabled": false
+               }
+
+   .. step:: Remove the following settings in the configuration file.
+
+      - ``connectionStrings``
+      - ``id``
+      - ``replicationSpecs.id``
+      - ``createDate``
```

Comment:
[q] was this necessary? when we should be doing this for the users, let me know if this is the case as we should then fix this

Reply:
It worked without removing these settings. I will remove these step.